django/contrib/messages/models.py

```diff
@@ -0,0 +1 @@
+# Necessary so that tests load the test models
```

Comment:
probably this file is not required anymore

Reply:
If I remove it and run the tests, I get

> django.db.utils.OperationalError: no such table: messages_someobject

I could not figure out how to create the tables when the tests run. Please let me know what is the proper way to do this.